docs/geninfo.rst

```diff
@@ -31,6 +31,20 @@ It also provides an initial implementation of a publisher and a subscriber using
 Compile
 -------
 
-In order to compile *fastrtpsgen* you first need to have `gradle <https://gradle.org/install>`_ and `java JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ installed (please, check the JDK recommended version for the gradle version you have installed).
+In order to compile *fastrtpsgen* you first need to have `gradle <https://gradle.org/install>`_ and
+`java JDK <http://www.oracle.com/technetwork/java/javase/downloads/index.html>`_ installed
+(please, check the JDK recommended version for the gradle version you have installed).
 
-To generate *fastrtpsgen* you will need to add the argument ``-DBUILD_JAVA=ON`` when calling CMake.
+If you want to compile *fastrtpsgen* java application, you will need to download its source code from
```

Comment:
¿"If you want"? If the user is reading this second paragraph, sure he want to compile it :). I think you can remove those words.

Reply:
```suggestion
To compile *fastrtpsgen* java application, you will need to download its source code from
```